EventsExpress/Mapping/AutoMapperProfile.cs

```diff
@@ -19,8 +19,12 @@ public AutoMapperProfile()
 
             #region USER MAPPING
             CreateMap<User, UserDTO>()
+                .ForMember(dest => dest.Id, opts => opts.MapFrom(src => src.Id))
+                .ForMember(dest => dest.Email, opts => opts.MapFrom(src => src.Email))
+                .ForMember(dest => dest.RoleId, opts => opts.MapFrom(src => src.RoleId))
```

Comment:
We don't need this. Properties with same names will be mapped by default

Reply:
deleted